README.md

```diff
@@ -240,7 +240,7 @@ Kubernetes:
 - Content of the shortcode:
 
   ```
-  <a href="https://kubernetes.io/docs/imported/release/notes/">1.8</a>
+  1.8
   ```
 - Usage in a document:
 
```

Comment:
Nit: Since you added the word "version" after "Kubernetes" in the actual documentation, it may be nice to also add in the example below (line 248 of this file): ` Kubernetes version {{% kubernetes-min-version %}}`

Reply:
Nice! Done.